scripts/prepare/bundle.ts

```diff
@@ -15,7 +15,7 @@ import { exec } from '../utils/exec';
 type Formats = 'esm' | 'cjs';
 type BundlerConfig = {
   entries: string[];
-  untypedEntries: string[];
```

Comment:
Looks like this just wasn't being used previously?

Reply:
I shortly used this when doing the preview prebundling, then found a better way, just never removed this feature from the bundling code.